public/js/app/templates/sessionNewTemplate.handlebars

```diff
@@ -28,6 +28,9 @@
       {{#if errors}}
         {{#link-to "forgot-password"}}Forgot your password?{{/link-to}}
       {{/if}}
+
+      <p>New to freefeed? <a href="https://docs.google.com/forms/d/1P4cyYUZlxABbWvZrM5jHRcMUFQFpc8qfbZWjGulobsA/viewform?c=0&w=1
```

Comment:
Page header uses "freefeed". I would hate to see "freefeed" and "FreeFeed" on the same page.